packages/client/src/components/Star/Star.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component }  from 'react';
+import classnames from 'classnames';
+
+import './Star.css';
+
+type StarProps = {
+    top?: string;
+    right?: string;
+    bottom?: string;
+    left?: string;
+    className?: string;
+    size?: 'small' | 'normal' | 'big';
+};
+
+export class Star extends Component<StarProps> {
+    render() {
+        let size = '';
+        if(this.props.size) {
+            size = this.props.size;
+        }
+        else {
+            const randomNum = Math.floor(Math.random() * (3) + 1);
+            if(randomNum === 1) {
+                size = 'small';
+            } else if(randomNum === 2) {
+                size = 'normal';
+            } else {
+                size = 'big';
+            }
+        }
+        const classNames = classnames('star', 'star__' + size, this.props.className);
```

Comment:
Вместо конкатенации лучше использовать шаблонную строку: они легче читаются и, если верить некоторым бенчмаркам, исполняются быстрее.

```suggestion
        const classNames = classnames('star', `star_size_${size}`, className);
```